script/release-artifact-cleanup.js

```diff
@@ -82,29 +86,39 @@ async function deleteTag (tag, targetRepo) {
       repo: targetRepo,
       ref: `tags/${tag}`
     })
-    console.log(`Successfully deleted tag ${tag} from ${targetRepo}`)
+    console.log(`${pass} successfully deleted tag ${tag} from ${targetRepo}`)
   } catch (err) {
-    console.log(`Couldn't delete tag ${tag} from ${targetRepo}: `, err)
-    process.exit(1)
+    console.log(`${fail} couldn't delete tag ${tag} from ${targetRepo}: `, err)
   }
 }
 
 async function cleanReleaseArtifacts () {
-  const releaseID = args.releaseID
+  const releaseId = args.releaseID.length > 0 ? args.releaseID : null
   const isNightly = args.tag.includes('nightly')
 
-  if (isNightly) {
-    await deleteDraft(releaseID, 'nightlies')
-    await deleteTag(args.tag, 'nightlies')
-  } else {
-    console.log('we are here')
-    await deleteDraft(releaseID, 'electron')
+  if (releaseId) {
+    if (isNightly) {
+      const deletedNightlyDraft = await deleteDraft(releaseId, 'nightlies')
+      // don't delete tag unless draft deleted successfully
+      if (deletedNightlyDraft) {
+        await Promise.all([
+          deleteTag(args.tag, 'electron'),
+          deleteTag(args.tag, 'nightlies')
+        ])
```

Comment:
If either of these two Promises reject, the entire `cleanReleaseArtifacts()` function will throw, keeping the bump commit from being reverted. Could we push these into an array or something and await them at the end?